modules/manage/partials/recovery-mode.adoc

```diff
@@ -0,0 +1,382 @@
+Recovery mode allows you to repair and restore a failed cluster that cannot start normally due to issues such as a system crash or out-of-memory (OOM) errors. In recovery mode, Redpanda limits functionality to cluster configuration changes and other manual administrative actions so that you can repair the cluster.
```

Comment:
```suggestion
Recovery mode allows you to repair and restore a failed cluster that cannot start normally due to issues such as  system crash or out-of-memory (OOM) errors. In recovery mode, Redpanda limits functionality to cluster configuration changes and other manual administrative actions so that you can repair the cluster.
```

Reply:
Because you are using plurals here, removed the "a", which suggests singular...doesn't match.

Reply:
Made crash plurla